internal/convert/id.go

```diff
@@ -0,0 +1,14 @@
+package convert
+
+import (
+	"crypto/sha1"
+	"fmt"
+)
+
+var h = sha1.New()
+
+func MakeId(s []byte) string {
```

Comment:
for public functions, Go requires descriptions, e.g.

// MakeId makes a unique ID using SHA